DockerComposeFixture/Compose/DockerCompose.cs

```diff
@@ -12,10 +12,15 @@ namespace DockerComposeFixture.Compose
 {
     public class DockerCompose : IDockerCompose
     {
-        public DockerCompose(ILogger[] logger)
+        private readonly IEnumerable<KeyValuePair<string, object>> environmentVariables;
+
+        public DockerCompose(ILogger[] logger, IEnumerable<KeyValuePair<string, object>> environmentVariables)
```

Comment:
Thanks for contributing! Why is this an IEnumerable<KeyValuePair<,>> instead of a IDictionary<,>? If it is so that the order of the env vars can be preserved then does the order matter?